process.md

```diff
@@ -61,126 +62,331 @@ This is for two main reasons:
 2. It's easier/cheaper to make changes before implementation (and you'll feel
    less emotionally invested!)
 
-In general, you should follow the [Tekton Enhancement Proposals
-(`TEP`)](./teps/) process. A Tekton Enhancement Proposal (TEP) is a
-way to propose, communicate and coordinate on new efforts for the
-Tekton project.  You can read the full details of the project in
+In general, you should follow the
+[Tekton Enhancement Proposals (`TEP`)](./teps/) process. A Tekton Enhancement
+Proposal (TEP) is a way to propose, communicate and coordinate on new efforts
+for the Tekton project. You can read the full details of the project in
 [TEP-1](./teps/0001-tekton-enhancement-proposal-process.md).
 
 Some suggestions for how to do this:
 
-1. Write up a design doc and share it with [the mailing list](contact.md#mailing-list).
-2. Bring your design/ideas to [our working group meetings](working-groups.md) for
-   discussion.
-3. Write a [`TEP`](./teps/) from the initial design doc and working
-   group feedback.
+1. Write up a design doc and share it with
+   [the mailing list](contact.md#mailing-list).
+2. Bring your design/ideas to [our working group meetings](working-groups.md)
+   for discussion.
+3. Write a [`TEP`](./teps/) from the initial design doc and working group
+   feedback.
 
 A great proposal will include:
 
-* **The use case(s) it solves** Who needs this and why?
-* **Requirements** What needs to be true about the solution?
-* **Alternative proposals** Even if alternatives aren't obvious,
-  forcing yourself to brainstorm a couple more approaches may give you
-  new ideas or make clear that your initial proposal is the best one
+- **The use case(s) it solves** Who needs this and why?
+- **Requirements** What needs to be true about the solution?
+- **Alternative proposals** Even if alternatives aren't obvious, forcing
+  yourself to brainstorm a couple more approaches may give you new ideas or make
+  clear that your initial proposal is the best one
 
 Also feel free to reach out to us on [slack](contact.md#slack) if you want any
 help/guidance.
 
 Thanks so much!!
 
-## OWNERS and Collaborators
+## Contributor Ladder
+
+<!--Big thanks to the folks at https://github.com/cncf/project-template
+for providing the base framework for this section.-->
+
+This contributor ladder outlines the different contributor roles within the
+project, along with the responsibilities and privileges that come with them.
+Community members generally start at the first levels of the "ladder" and
+advance up it as their involvement in the project grows. Our project members are
+happy to help you advance along the contributor ladder.
+
+Each of the contributor roles below is organized into lists of three types of
+things. "Responsibilities" are things that contributor is expected to do.
+"Requirements" are qualifications a person needs to meet to be in that role, and
+"Privileges" are things contributors on that level are entitled to.
```

Comment:
nit: could you add a list here that lists the roles in the ladder below?

Reply:
not important enough to block the PR on but pinging this again @wlynch

Reply:
Done.